Resources/Prototypes/_LostParadise/Entities/Objects/Specific/halloween.yml

```
@@ -0,0 +1,35 @@
+﻿- type: entity
+  name: paper bag
+  id: LPPPaperBagCandy
+  parent: CandyBucket
+  description: A bag for storing chemistry products, such as pills, pill canisters, bottles, and syringes.
+  components:
+    - type: Sprite
+      sprite: _LostParadise/Objects/Specific/halloween.rsi/paper_bag.rsi
+      layers:
+      - state: empty_icon
```

yamllint

[error] wrong indentation: expected 8 but found 6

(indentation)